ruby/global-data-variable.md

```diff
@@ -0,0 +1,63 @@
+# Global `DATA` variable
```

Comment:
`DATA` is a constant, not a global variable. Thoughts on changing the title to something along the lines of: `# The DATA constant`?